src/proof/dot/dot_printer.h

```diff
@@ -87,10 +138,57 @@ class DotPrinter
 
   /** Traverse proof node and populate d_lbind
    *
-   * @param pn the proof node to be traversed
+   * @param pn The proof node to be traversed
    */
   void letifyResults(const ProofNode* pn);
 
+  /** Define the proof node type and populate d_nodesClusterType and
+   * d_scopesArgs.
+   * @param pn The proof node to categorized.
+   * @param last The type of the parent of the current proof node
+   * @return the current node type
+   */
+  ProofNodeClusterType defineProofNodeType(const ProofNode* pn,
+                                           ProofNodeClusterType last);
+
+  /** Verify if the proof node is an input node. An input node is a proof node
```

Comment:
```suggestion
  /** Whether the proof node is an input node. 
   *
   * An input node is a proof node
```

Change below to follow this pattern is well.

"Verified" we generally use in a more "check with respect to this specification" sense.